src/ToolBox/SOS/lldbplugin/CMakeLists.txt

```diff
@@ -76,6 +76,7 @@ endif()
 # If the file in a directory is found the result is stored in the variable and the search will not be repeated unless the variable is cleared.
 find_path(LLDB_H "lldb/API/LLDB.h" PATHS "${WITH_LLDB_INCLUDES}" NO_DEFAULT_PATH)
 find_path(LLDB_H "lldb/API/LLDB.h")
+find_path(LLDB_H "lldb/API/LLDB.h" PATHS "/usr/lib/llvm-6.0/include")
 find_path(LLDB_H "lldb/API/LLDB.h" PATHS "/usr/lib/llvm-4.0/include")
```

Comment:
Should this also include llvm-5.0/include? Same for above in `find_library` call.
Btw, did you get LLDB.h? I almost installed all the 6.0 packages but couldn't find it. Do you know which package installed LLDB.h?

Reply:
I built with `./build.sh cmakeargs -DENABLE_LLDBPLUGIN=0` because of absence of LLDB.h

Reply:
I have looked at the build log and it has actually picked LLDB.h from an older installed version. I've tried to install all the packages available for llvm 6.0 and that version of the header didn't get installed. I don't know if that's an intention or mistake on the llvm side. The llvm 5.0 still contains it.

> Should this also include llvm-5.0/include

I haven't originally added it since there used to be  a bug in clang 5.0 preventing compilation of our stuff. I have thought that it was not fixed yet, but I've just tried the 5.0 package and it worked. So I am going to augment this change to add support for 5.0 too so that we don't skip that version.

Reply:
That bug was fixed in 5.0.1 😄
I asked about it today in #lldb channel over at IRC, maybe @krytarowski can provide some insights on why LLDB.h is missing from https://apt.llvm.org liblldb-6.0-dev package. Is it a bug in packaging since it is available in release_60 branch https://github.com/llvm-mirror/lldb/blob/release_60/include/lldb/API/LLDB.h?

Reply:
I don't use apt.llvm.org.